ansible-role-requirements.yml

```diff
@@ -18,57 +18,3 @@
   scm: git
   src: https://github.com/openstack/openstack-ansible-os_octavia.git
   version: master
-# TODO(odyssey4me):
-# Remove this once this SHA is included in our OSA SHA
-- name: plugins
-  scm: git
-  src: https://git.openstack.org/openstack/openstack-ansible-plugins
-  version: d9669419bf63d92c7fd0702ee6ca48fab3ea0a0a
-# TODO(odyssey4me)
-# Remove this once this SHA is included in our OSA SHA
-- name: lxc_hosts
-  scm: git
-  src: https://git.openstack.org/openstack/openstack-ansible-lxc_hosts
-  version: 09a9ff6297038f4eba49c4cac31ca78f4abb9d51
-# TODO(odyssey4me)
-# Remove this once this SHA is included in our OSA SHA
-- name: rabbitmq_server
-  scm: git
-  src: https://git.openstack.org/openstack/openstack-ansible-rabbitmq_server
-  version: 6546786a28ad5e28a9158c1eb3912915b427e9f7
```

Comment:
As you will bump the artifacts, you'll also probably bump erlang/rabbitmq (to verify what we have defined in our apt artifact sources).

This also means we won't have the pin in b8f4fe98998268197782603f029ae1cb29e5b8c5.

Did you foresee that, or should I investigate further?

Reply:
The erlang patches are not yet included in the integrated build for OSA, so we don't need any changes.

Reply:
Yes, I agree, but I guess they were included in the role's for a reason...

Reply:
As you heard in the discussion yesterday (or was it the day before), there's some dispute about whether they're the right thing to use... so at this stage I'd rather let it play out. Also, the less tech debt we carry, the better.